index.md

```diff
@@ -29,22 +18,29 @@ Along with bug fixes and general improvements, Anbox Cloud 1.13 comes with:
 * Direct rendering for Intel and AMD GPUs
 * OpenGL ES 3.2 support
 
-|  |  |
-|--|--|
-| [Release notes](https://discourse.ubuntu.com/t/release-notes/17842) | All new features, improvements and bug fixes |
-| [Release roadmap](https://discourse.ubuntu.com/t/release-roadmap/19359) | Planned updates and features for upcoming releases |
+For detailed information, see the [Release notes](https://discourse.ubuntu.com/t/release-notes/17842).
+
+## Project and community
+
+Anbox Cloud is a Canonical product. It is closed source and has no link to the Anbox open source project.
```

Comment:
I'd suggest:

```
## Development, support and user community

Anbox Cloud is a Canonical product.
```

To say "It is closed source [btw: 'proprietary' would be better] and has no link to the Anbox open source project" seems somehow a bit negative. Personally, I would not be inclined to say it. However use your judgement about whether this clarification is required.

Reply:
Discussed offline with @evildmp - we should keep it in this section and not move it to the intro. But we should word it more positively - something like "Anbox Cloud grew out of the Anbox open-source project. Its code base is now completely independent."
Does that sound better?